specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2019-08-01/cosmos-db.json

```diff
@@ -4592,23 +4592,27 @@
       "description": "The properties of an Azure Cosmos DB SQL database",
       "type": "object",
       "properties": {
-        "_colls": {
-          "type": "string",
-          "description": "A system generated property that specified the addressable path of the collections resource."
-        },
-        "_users": {
-          "type": "string",
-          "description": "A system generated property that specifies the addressable path of the users resource."
-        }
-      },
-      "allOf": [
-        {
-          "$ref": "#/definitions/SqlDatabaseResource"
-        },
-        {
-          "$ref": "#/definitions/ExtendedResourceProperties"
+        "resource": {
```

Comment:
Is adding 'resource' around all these properties a change to your service or a correction to the swagger documentation?

Reply:
It's the fix to our service. As the Export template in portal is broken.

Reply:
Adding the "resource" is the the change to this swagger documentation. We shouldn't have response body structure differentiate from the request.